wiki/Glossary/Changelog/en.md

```diff
@@ -1,13 +1,35 @@
 ---
-stub: true
 tags:
   - history
 ---
 
 # Changelog
 
+<!-- for the purpose of learning the history through the Wayback Machine (https://web.archive.org/), the changelog is available at different URLs:
+  - http://osu.ppy.sh/?p=changelog
+  - http://osu.ppy.sh/p/changelog
+--->
+
 The [changelog](https://osu.ppy.sh/home/changelog) is a page where peppy and the osu!dev team detail their day-by-day improvements, additions, and bug fixes to the game. All changes from prior releases are recorded here for future browsers to view.
 
```

Comment:
we can add something like that too, i guess?

```suggestion

The changelogs for the website and osu!(lazer) are generated automatically based on GitHub releases of those projects, while the rest are maintained manually.
```

Reply:
absolutely